wpimath/src/main/native/include/frc/geometry/Translation2d.h

```diff
@@ -73,6 +82,13 @@ class WPILIB_DLLEXPORT Translation2d {
    */
   constexpr units::meter_t Y() const { return m_y; }
 
+  /**
+   * Returns a vector representation of this translation.
+   *
+   * @return A Vector representation of this translation.
+   */
+  Eigen::Vector2d ToVector() const;
```

Comment:
This can be constexpr if you use the init list constructor (e.g., `return Eigen::Vector2d{{m_x.value(), m_y.value()}};`).